test/unit/c_api/eigensolver/test_gen_eigensolver_c_api_wrapper.h

```diff
@@ -20,78 +20,171 @@ DLAF_EXTERN_C int C_dlaf_symmetric_generalized_eigensolver_s(
     const int dlaf_context, const char uplo, float* a, const struct DLAF_descriptor desca, float* b,
     const struct DLAF_descriptor descb, float* w, float* z, const struct DLAF_descriptor descz);
 
+DLAF_EXTERN_C int C_dlaf_symmetric_generalized_eigensolver_partial_spectrum_s(
+    const int dlaf_context, const char uplo, float* a, const struct DLAF_descriptor desca, float* b,
+    const struct DLAF_descriptor descb, float* w, float* z, const struct DLAF_descriptor descz,
+    const SizeType eigenvalues_index_begin, const SizeType eigenvalues_index_end);
+
```

Comment:
These definitely look like they'd enjoy being generated with so many variants, but I suppose this might be the last major change in the API for a while? Definitely not asking you to change it in this PR, if at all, just thinking out loud...

Reply:
YES! I think this is the very last thing I will add manually: if we need to add something else, then this definitely needs to be generated.

I might already try to generate the Fortran interface with FYPP (which I already use for tests).